src/game_battlealgorithm.cpp

```diff
@@ -1439,6 +1441,8 @@ bool Game_BattleAlgorithm::Skill::IsReflected() const {
 					}
 				}
 			} else {
+				// Return false if no target exist
```

Comment:
How about just `else if (GetTarget() != nullptr)` and then fall through if no targets

Reply:
Your proposal works for me. I changed the code according to your suggestion.